src/en/clouds.md

```diff
@@ -1,17 +1,18 @@
-Title: Juju clouds
-TODO: Needs to explain available auth types for clouds
+Title: Clouds
+TODO:  Needs to explain available auth types for clouds
   
 # Clouds
 
-Juju ships with the ability to seamlessly use a number of public clouds
-(including Amazon Web Services, Azure, Google Compute Engine, Joyent and
-Rackspace) to deploy workloads, as well as private clouds (e.g.
-OpenStack) which you configure.
+Juju has built-in support for all major public clouds such as AWS (Amazon),
```

Comment:
You need to stop conflagrating everything. These paragraphs do not need to be touched in the context of 'update juju status'

Reply:
I know. :) It's a weakness of mine.

Reply:
One consequence of changing this is that, as you are altering the 2.2 branch, these changes are not reflected in master.  It doesn't matter for the status stuff which is due to change in 2.3 anyhow, but it does matter for future changes which will now cause conflicts